app/views/layouts/application.html.erb

```diff
@@ -53,12 +53,16 @@
         </form> -->
         <ul class="nav navbar-nav navbar-right">
           <% if current_user %>
+            <% unless current_user.subscribed %>
```

Comment:
Using unless/else is an anti-pattern.  Use if/else and reverse the order, unless there's a really good reason not to.